pkg/roles/rolemanager/api/requests.go

```diff
@@ -53,6 +53,52 @@ func (req listRolesReq) validate() error {
 	return nil
 }
 
+type listEntityMembersReq struct {
```

Comment:
Are members = users who can access it? If yes, please change to `listEntityUsersReq` and add a comment explaining that.